projects/observability/src/pages/explorer/explorer-service.test.ts

```diff
@@ -52,14 +48,25 @@ describe('Explorer service', () => {
     })
   }));
 
-  test('creates nav params correctly', () => {
+  const createService = createServiceFactory({
+    service: ExplorerService,
+    providers: [
+      mockProvider(FilterBuilderLookupService, { lookup: lookupMock }),
+      mockProvider(MetadataService, { getAttribute: getAttributeMock }),
+      mockProvider(TimeRangeService, {
+        toQueryParams: () => ({
+          ['time']: new FixedTimeRange(
+            new Date('2019-09-19T16:40:45.141Z'),
+            new Date('2019-09-19T16:55:45.141Z')
```

Comment:
nit: this seems the same below, and can be converted into a const

Reply:
Oh yeah, but since there's just two instances I left it as it is